test/endpoints/automations.test.js

```diff
@@ -20,4 +20,17 @@ describe('Tests for automation endpoints\n', () => {
         done();
       });
   });
+
+  it('Should return a 500 server error', (done) => {
+    chai
+      .request(app)
+      .get('/api/v1/automations')
```

Comment:
Great job @Nerldy. I am curious, why is this an error? And why a 500 error?

Reply:
Thanks. It was to catch an error if the url didn't have the correct queries but thanks to @valmayaki suggestion I've gone instead with a default value and removed the catch function.